client/src/redux/actions/appInitializationAction.js

```diff
@@ -10,8 +10,8 @@ export const initializeApp = async dispatch => {
   const location = `${currentLocation.coords.latitude},${currentLocation.coords.longitude}`;
 
   await getAndSetAccessToken();
-  dispatch(getLocationAction(currentLocation));
-  dispatch(getWeatherAction(location));
-  dispatch(getHourlyWeatherAction(location));
-  dispatch(getDailyWeatherAction(location));
+  await dispatch(getLocationAction(currentLocation));
```

Comment:
1) I don't think this change has something to do with this PR's topic
2) I don't see that any of subsequent requests is using data from previous one. Then the question is why not Promise.all ?